pages.es/common/quarkus.md

```diff
@@ -0,0 +1,36 @@
+# quarkus
+
+> Crea proyectos Quarkus, gestiona extensiones y realiza tareas esenciales de compilación y desarrollo.
+> Más información: <https://quarkus.io/guides/cli-tooling>.
+
+- Crea un nuevo proyecto de aplicación en un directorio nuevo:
+
+`quarkus create app {{nombre_del_proyecto}}`
+
+- Ejecuta el proyecto actual en el modo de codificación en vivo:
+
+`quarkus dev`
+
+- Ejecuta la aplicación:
+
+`quarkus run`
+
+- Ejecuta el proyecto actual en modo de prueba continua:
+
+`quarkus test`
+
+- Añade una o más extensiones al proyecto actual:
+
+`quarkus extension add {{nombre_extensión_1 nombre_extensión_2 ...}}`
```

Comment:
```suggestion
`quarkus extension add {{nombre_extensión1 nombre_extensión2 ...}}`
```
The English page is wrong (sorry, my fault), we don't add an underscore before a number in theses cases.

We need to update both pages.

Reply:
No problem, I probably might have committed the same mistake.